packages/web-react/src/components/Dropdown/README.md

```diff
@@ -123,7 +123,7 @@ import { UncontrolledDropdown, DropdownTrigger, DropdownPopover } from '@lmc-eu/
 | ----------------- | ------------------------------------------------ | -------------- | -------- | ---------------------------------------------- |
 | `enableAutoClose` | `bool`                                           | `true`         | ✕        | Enables close on click outside of Dropdown     |
 | `fullWidthMode`   | [`DropdownFullwidthMode`][dropdownfullwidthmode] | `off`          | ✕        | Full-width mode                                |
-| `id`              | `string`                                         | -              | ✔        | Component id                                   |
+| `id`              | `string`                                         | —              | ✔        | Component id                                   |
```

Comment:
What have changed? I think the new one is "longer" a little bit...

Reply:
there was a different type of dash. In my IDE it was more visible. The new one is the one we use everywhere. 
old -
new —